pkg/installrpm.sh

```diff
@@ -23,8 +23,8 @@ cat INSTALLED_FILES.backup|egrep -v 'gempython/__init__.py*' > INSTALLED_FILES
 # set permissions
 cat <<EOF >>INSTALLED_FILES
 %attr(0755,root,root) /opt/cmsgemos/bin/*.py
-%attr(0755,root,root) /usr/lib/python*/site-packages/gempython/scripts/*.py
-%attr(0755,root,root) /usr/lib/python*/site-packages/gempython/gemplotting/macros/*.py
+%attr(0755,root,root) /opt/cmsgemos/lib/python*/site-packages/gempython/scripts/*.py
```

Comment:
This was also explicitly avoided in the initial setup, as in future versions this will be replaced with system site-packages location macros provided by `rpm` (currently only works for `cc7`)

Reply:
> system site-packages location macros provided by `rpm`
Could you provide more details about this? A link to the `rpm` doc about this would be great